go.mod

```diff
@@ -60,20 +67,12 @@ replace (
 	github.com/mitchellh/go-homedir => github.com/mitchellh/go-homedir v1.1.0
 	github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.7.0
 	github.com/onsi/gomega => github.com/onsi/gomega v1.4.3
-	github.com/openshift/api => github.com/openshift/api v3.9.0+incompatible
-	github.com/openshift/client-go => github.com/openshift/client-go v3.9.0+incompatible
 	github.com/sirupsen/logrus => github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/cobra => github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag => github.com/spf13/pflag v1.0.3
 	github.com/spf13/viper => github.com/spf13/viper v1.4.0
 	github.com/stretchr/testify => github.com/stretchr/testify v1.3.0
 	gopkg.in/inf.v0 => gopkg.in/inf.v0 v0.9.1 // indirect
-	k8s.io/api => k8s.io/api v0.0.0-20190313235455-40a48860b5ab
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20190315093550-53c4693659ed
-	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190313205120-d7deff9243b1
-	k8s.io/client-go => k8s.io/client-go v11.0.0+incompatible
-	k8s.io/klog => k8s.io/klog v0.4.0 // indirect
-	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20190722073852-5e22f3d471e6 // indirect
-	k8s.io/utils => k8s.io/utils v0.0.0-20190809000727-6c36bc71fc4a // indirect
+	helm.sh/helm/v3 => helm.sh/helm/v3 v3.1.0-rc.1.0.20200219163831-598dc504d634
```

Comment:
can we pin this to an actual release rather than a rc, they do have v3.1.0 now: https://github.com/helm/helm/releases

Reply:
@yashbhutwala because of an error propagation message issue, we are pinned to rc release. once they released, we will point to actual release version